README.md

```diff
@@ -10,78 +10,66 @@
-
-```sh
-cd shifu
-kubectl apply -f pkg/k8s/crd/install/shifu_install.yml
-```
 
-### Try it out
+Shifu is a Kubernetes-native IoT development framework that 
 
-We present you [***Shifu*** Demo](https://shifu.run/disclaimer). 
+greatly improves the efficiency, quality and reusability of IoT application development.
 
-The Demo will show you how ***Shifu*** creates and manages digital twins of any physical devices.
 
-### Documentation
-
-For more information please visit our [documentation](https://shifu.run/docs/).
-
-## Design principle
-
-Our job is to make developers and operators happy. Which is why all our designs are developer-oriented:
-
-### 📡Easy to deploy
+|Feature|Description |
+|---|---|
+|🔌 Fast Device integration &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|Compatible with almost all protocols and drivers.|
+|👨‍💻 Efficient Application development|Shifu structually virtualizes each device and expose its capabilities in the form of APIs.|
+|🔧 Easy Operation & Maintenance|Kubernetes-native framework, sparing the need for maintaining an additional O&M infrastructure.|
+</div>
+<br/><br/>
 
-Shifu can be deployed with a single command.
+# 🪄Demo
+<div align="center">
+<img width="900px" src="./img/demo-camera.gif"></img>
+<img width="900px" src="./img/demo-plc.gif"></img>
+</div>
+<br/><br/>
 
-### 🤖Plug'n'Play
+# 🔧Install
 
-Shifu will automatically recognize and provide basic functionalities to a supported IoT device. Once the developer completes Shifu's template, all features of the device should be immediately available.
+- If you have started a Kubernetes cluster, use the command `kubectl apply` to install Shifu in your cluster:
 
-### 🪄High extensibility
+    ```sh
+    cd shifu
+    kubectl apply -f pkg/k8s/crd/install/shifu_install.yml
+    ```
 
-Developer can further implement Shifu's interface/SDKs to create custom features and unleash endless possibilities.
+- If you don't have a Kubernetes cluster on your computer, that's totally ok, you can download our demo to try it out.
+  - Download Docker
+  
+    [Mac](https://docs.docker.com/desktop/install/mac-install/) | [Windows(WSL)](https://docs.docker.com/desktop/install/windows-install/) | [Linux](https://docs.docker.com/desktop/install/linux-install/)
+  - Download Shifu Demo with a single command
+    ```sh
+    curl -sfL https://raw.githubusercontent.com/Edgenesis/shifu/main/test/scripts/shifu-demo-install.sh | sudo sh -
+    ```
 
-### 🔧Zero maintenance
+- Now that you have installed Shifu, please visit our🗒️[documentation](https://shifu.run/docs/) to🔌[connect a device](https://shifu.run/docs/guides/cases/) and 👨‍💻[develop your own application](https://shifu.run/docs/guides/application/)!
 
-Shifu aims to achieve zero maintenance by adopting cutting-edge cloud native best practices. After all, Shifu should take care of himself and make DevOps' lives easier!
+# 💖Community
```

Comment:
leave a space for this icon and text too if that looks better? so do other icons.

Reply:
ok, are the icons appear too flashy or not? do you think it's all right?

Reply:
I think the license icon seems not so related.
The icons for the features table better represent  "Fast" "Efficient" "Easy" rather than the object?

others are ok

Reply:
there's no icon for "efficient" and "easy" :)

Reply:
the 3rd icon looks duplicated with the icon for Install, better choose another one?

Reply:
👨‍🔧how about this one